### PR TITLE
Onboard CCR performance test to new build.ci.opensearch.org

### LINKED ISSUE #3178: Move the CCR performance tests to the new build.ci.opensearch.org
We have a periodic performance run for CCR to benchmark the replication performance. We need to move the workflow to the new setup i.e. buld.ci.opensearch.org

### PR DESCRIPTION
**Pull request by mohitamg**
Signed-off-by: Mohit Kumar <mohitamg@amazon.com>

### Description
This PR contains the necessary changes related to triggering the CCR performance test and updation of parameters to be taken for running the tests.

### Issues Resolved
Closes #3178 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: jenkins/cross-cluster-replication/perf-test.jenkinsfile [hunk 1]
```diff
@@ -14,6 +14,28 @@ pipeline {
         BUNDLE_MANIFEST = 'bundle-manifest.yml'
         JOB_NAME = 'ccr-perf-test'
     }
+    triggers {
+        parameterizedCron '''
+            H 9 * * * %BUNDLE_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.2.1/latest/linux/arm64/tar/dist/opensearch/manifest.yml;TEST_ITERATIONS=3;WARMUP_ITERATIONS=2
+            H 9 * * * %BUNDLE_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.2.1/latest/linux/x64/tar/dist/opensearch/manifest.yml;TEST_ITERATIONS=3;WARMUP_ITERATIONS=2
+            H 9 * * 3 %BUNDLE_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.2.1/latest/linux/arm64/tar/dist/opensearch/manifest.yml;TEST_ITERATIONS=250;WARMUP_ITERATIONS=0
```
Comment (kotwanikunal) [inline jenkins/cross-cluster-replication/perf-test.jenkinsfile:21]: Can we skip longevity tests for now? We can start with the basics and then eventually start adding versions and long running test in.

**Comment by kotwanikunal** [comment]
@mohitamg Groovy tests are failing. You might have to run `./gradlew test -info -Ppipeline.stack.write=true`. 
More on this here - https://github.com/opensearch-project/opensearch-build/blob/main/DEVELOPER_GUIDE.md#jenkins-pipelines-and-shared-libraries

**Review by kotwanikunal — APPROVED** [review]

**Comment by kotwanikunal** [comment]
@opensearch-project/engineering-effectiveness Can we get some 👀  on this PR?

**Review by prudhvigodithi — APPROVED** [review]

**Comment by prudhvigodithi** [comment]
LGTM!
adding @gaiksaya and @peterzhuamazon

**Review by gaiksaya — APPROVED** [review]
LGTM. Just checking if 2.4.0 is the intended version for performance testing?

**Comment by mohitamg** [comment]
Yes @gaiksaya , as a starting point I am keeping 2.4.0 only for performance testing, if everything is fine , then will add other versions as per necessity.

**Pull request merged**